include/WCSimPrimaryGeneratorAction.hh

```diff
@@ -42,8 +45,8 @@ class WCSimPrimaryGeneratorAction : public G4VUserPrimaryGeneratorAction
 
         // These go with jhfNtuple
         G4int GetVecRecNumber(){return vecRecNumber;}
-        //G4int GetMode() {return mode;};
-        InteractionType_t GetMode() {return mode;};
+        G4int GetMode() {return mode;};
+        //InteractionType_t GetMode() {return mode;};
```

Comment:
@akutsuR and @nickwp , do you need that GetMode returns and "InteractionType_t" for E61

Reply:
@bquilain and @nickwp I have never used it for my IWCD analyses and the IWCD analysis codes used for the XSec analyses also does not use it. (We have used copy of NEUT output, which is produced in WCSim)

Reply:
I also have never used this. It looks like this change was introduced in [this commit](https://github.com/nuPRISM/WCSim/commit/ac4d7078c105e0949e48ff27775a9b70a6053677) by Tom Feusels in 2016, so it's been there for a long time and possibly some old analysis code uses it. Perhaps @tfeusels or @mscott201 would know?

Reply:
(I note that the commit comment says that it "breaks backwards compatibility for Nuance, which is deprecated anyway" ... but five years later nuance input is still being used in some cases)